src/client/containers/CreateTeamPage/CreateTeamPage.component.js

```diff
@@ -1,6 +1,29 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import HeaderTitle from '../../components/Title/Title';
+import Footer from '../../components/FooterComponent/FooterComponent';
+import SingleInputFormComponent from '../../components/SingleInputFormComponent/SingleInputFormComponent';
+import AddAvatarComponent from '../../components/AddAvatarComponent/AddAvatarComponent'
```

Comment:
Did you push the AddAvatarComponent?
The build failed because it can't resolve the component. The files is not on the PR.

Reply:
They are on this PR https://github.com/HackYourFuture-CPH/fp-class12/pull/171